src/status_im/contexts/wallet/send/input_amount/view.cljs

```diff
@@ -159,7 +160,23 @@
                                             [:show-bottom-sheet
                                              {:content (fn []
                                                          [select-asset-bottom-sheet
-                                                          clear-input!])}])]
+                                                          clear-input!])}])
+            loading-suggested-routes?     (rf/sub
+                                           [:wallet/wallet-send-loading-suggested-routes?])
+            sender-network-values         (rf/sub
+                                           [:wallet/wallet-send-sender-network-values])
+            suggested-routes              (rf/sub [:wallet/wallet-send-suggested-routes])
+            routes                        (when suggested-routes
+                                            (or (:best suggested-routes) []))
+            no-routes-found?              (and
+                                           (every? (fn [{:keys [total-amount]}]
```

Comment:
this condition can be extracted do a separate function for better readability, something like `every-network-value-is-zero`

Reply:
Done